README.md

```diff
@@ -1,24 +1,203 @@
+Markdown file's name).  For example, the section on Augury uses
+`Augury` as the title and `augury` as the slug.
+
+Regular slides are formatted as:
+
+~~~
+<!-- .slide: id="SLIDE_SLUG" -->
+## SLIDE_TITLE
+
+- Bullet
+- Bullet
+
+#####_path/to/example/file_
+```FILE_TYPE
+code sample
 ```
-cd slides
-npm install
-npm start
-```
+~~~
+
+`SLIDE_SLUG` must be a hyphenated unique identifier for this particular slide
+starting with the slug for the section.  For example, the first content slide
+in the section on Augury has `augury-installing` as its ID.  [Reveal.js][reveal]
+uses these IDs to create unique (bookmarkable) URLs for the slides, which in
+turn allows us to refer to particular slides when filing issues.
+
+The slide's title should be brief (no more than 3-4 words) and Title Cased.
+
+Every code sample should have an H5-level heading immediately in front
+of it with the path to the file containing the code in italics.  The
+code block introduced by triple backquotes should specify a file type
+for syntax highlighting, such as `ts` (for TypeScript), `js` (for plain
+old JavaScript), or `html` (for HTML).
+
+Slides with quiz questions should have the following header:
+
+~~~
+<!-- .slide: id="SECTION-quiz-NUMBER" -->
+<!-- .slide: data-background="../content/images/question-slide.jpg" -->
+
+## Quiz
+~~~
+
+The important differences between these slides and regular slides are
+the use of `quiz` and a sequence number in the ID, and the inclusion
+of the "question" slide background image.
+
+Every quiz should also have answers on a sub-slide.  To separate a
+sub-slide from a regular slide, use `+++` instead of `---`.  The
+sub-slide should then look like this:
+
+~~~
++++
+<!-- .slide: data-background="../content/images/answer-slide.jpg" -->
+
+## Answer
+~~~
+
+The first line after `+++` includes the "answer" background image.
+The sub-slide's title should always be "Answer".
+
+## For Each Training Class
+
+1.  Create a branch in this repository named after the start date of the training,
+    e.g., `2017-01-01` for a training class that starts on January 1, 2017.
+    (We name branches this way so that client names do not appear in the repository,
+    and to make it easier to track the evolution of material.)
 
-To see the slides in your browser, go to [http://localhost:8081](http://localhost:8081/#/)
+1.  Modify `./slides/src/content/introduction.md` to include the biographies of the instructors,
+    the class's schedule, and other specific information.
+    Please do *not* include these changes in any PRs you submit to improve the core material.
 
-To modify or add more diagrams go to [this](https://drive.google.com/drive/u/0/folders/0B6Cn_hQVgj2PRHZHQTFxd21UX00) Google Drive folder.
+1.  After starting the server, generate a PDF of your slides by going to
+    [http://localhost:8081/?print-pdf](http://localhost:8081/?print-pdf)
+    and using Google Chrome's print utility (Save as PDF).
```

Comment:
I might have missed it but was there a request from marketing regarding a PDF version or are we driving this? I haven't seen any demand for PDF versions of the slide content